server/api.go

```diff
@@ -87,6 +173,13 @@ func (a *API) processActivity(w http.ResponseWriter, req *http.Request) {
 	a.p.API.LogDebug("Change activity request", "activities", activities)
 	errors := ""
 	for _, activity := range activities.Value {
+		if activity.EncryptedContent.Data != "" {
+			content, err := a.processEncryptedContent(activity.EncryptedContent)
+			if err != nil {
+				a.p.API.LogError("Invalid encrypted content", "error", err)
```

Comment:
don't we need to add `continue` here if we were not able to process the content

Reply:
My way of thinking here is, I try to get the encrypted content, if it doesn't work, just keep going, it doesn't matter, because in that case you are going to try to use the "standard" method of getting the information from the server.

Reply:
1. By standard method you mean the webhook secret?
2. What are some cases where the encrypted content may be invalid but yet be a valid request overall?

Reply:
By standard method I mean, using the current approach. Trying to get the information from the server doing a API request by the user that belongs to the channel.

The most common case of encrypted content to be invalid is the absence of encrypted content entirely. If you are doing "request forgery" there, you can do it by cleaning up the encrypted content, instead of passing an invalid content enayway.

Reply:
And yes, in any case we are verifying the webhook secret.